source/common/grpc/BUILD

```diff
@@ -3,7 +3,6 @@ licenses(["notice"])  # Apache 2
 load(
```

Comment:
I don't think you need to revert this file?

Reply:
Yes, I can revert this file.  

My new change is: to making this google_grpc_context_lib using google_grpc,  not support "target:gce".   

But in source/exec/BUILD,use google_grpc_select to depend on google_grpc_context_lib
which will check if google_grpc is disbled or not.

My new change will be:

diff --git a/source/exe/BUILD b/source/exe/BUILD
index 719cded40..47256f5ae 100644
--- a/source/exe/BUILD
+++ b/source/exe/BUILD
@@ -8,6 +8,7 @@ load(
     "envoy_cc_posix_library",
     "envoy_cc_win32_library",
     "envoy_package",
+    "envoy_select_google_grpc",
 )
 load(
     "//source/extensions:all_extensions.bzl",
@@ -71,12 +72,12 @@ envoy_cc_library(
         "//source/common/api:os_sys_calls_lib",
         "//source/common/common:compiler_requirements_lib",
         "//source/common/common:perf_annotation_lib",
-        "//source/common/grpc:google_grpc_context_lib",
         "//source/common/stats:symbol_table_creator_lib",
         "//source/server:hot_restart_lib",
         "//source/server:hot_restart_nop_lib",
         "//source/server/config_validation:server_lib",
-    ] + select({
+    ] + envoy_select_google_grpc(["//source/common/grpc:google_grpc_context_lib"])
+      + select({
         "//bazel:disable_signal_trace": [],
         "//conditions:default": [
             "//source/common/signal:sigaction_lib",

Reply:
Here is my new PR: https://github.com/envoyproxy/envoy/pull/8409

Reply:
#8409 is closed, can you advise on what PR you want reviewed?

Reply:
@htuch could you help to just merge this "revert" PR?  I found out that I don't need to make any changes in upstream envoy.  I just need to change copybara config to fix target.gce issue.